seaport-subgraph/src/modules/airstack/nft.ts

```diff
@@ -0,0 +1,229 @@
+import {
+    Address,
+    BigDecimal,
+    BigInt,
+    dataSource,
+    Bytes,
+    log,
+} from "@graphprotocol/graph-ts";
+
+import {
+    AirAccount,
+    AirNftTransaction,
+    AirToken,
+    AirBlock,
+    AirNftSaleRoyalty
+} from "../../../generated/schema";
+
+export namespace nft {
+    export function trackNFTSaleTransactions(
+        txHash: string,
+        txIndex: BigInt,
+        NftSales: Sale[],
+        protocolType: string,
+        protocolActionType: string,
+        timestamp: BigInt,
+        blockHeight: BigInt,
+        blockHash: string
+    ): void{
+        if (NftSales.length == 0) {
+            return;
+        }
+
+
+        let chainID = 1;
+        let block = getOrCreateAirBlock(chainID.toString()+"-"+blockHeight.toString());
+        block.number = blockHeight;
+        block.timestamp = timestamp;
+        block.hash = blockHash;
+        
+        let transactionCount = NftSales.length;
+        for(let i=0; i<transactionCount; i++){
+            // Payment Token
+            let paymentToken = getOrCreateAirToken(NftSales[i].paymentToken.toHexString());
+            paymentToken.save();
+
+            // Account
+            let buyerAccount = getOrCreateAirAccount(NftSales[i].buyer.toHexString());
```

Comment:
lets add chainId to id.

Reply:
Should we also add chainID in the payment token id?